docs/developer.rst

```diff
@@ -149,6 +117,33 @@ After installing the servers we need to configure them by granting privileges to
   Create .pgpass in your home directory:
   localhost:*:testdb:postgres:testpass
 
+To run tests we use pytest.
+From the source top level directory, run
+
+.. code-block:: sh
+
+  $   py.test
+
+
+In case we want to run tests on a specific test category, we add the path of the test module, py.test [path]
```

Comment:
"In case we want to run tests on a specific test category, we add the path of the test module, py.test" -> "To run tests on a specific test category add the path of the test module to the end of the py.test command"